app/experimenter/experiments/forms.py

```diff
@@ -773,7 +774,10 @@ class ExperimentReviewForm(
     )
     review_qa_requested = forms.BooleanField(
         required=False,
-        label="QA Requested",
+        label=mark_safe(
+            f"QA <a href={settings.JIRA_URL} target='_blank'>"
+            "Jira</a> Request Sent"
```

Comment:
I bet you could recombine these back into one line now.

Reply:
it was still slightttlllyyyy too long I think.... I'll double check.

Reply:
I'm seeing 
`./experimenter/experiments/forms.py:778:80: E501 line too long (83 > 79 characters)` when trying to change it, so I think I'm slightly over the limit without two lines.

Reply:
Ah, my line-length limits are tuned to Normandy, where we use 99 chars instead of 79. Carry on.